Oracle/lab_result_cm.sql

```diff
@@ -157,14 +158,20 @@ select distinct cast(lab.LAB_RESULT_CM_ID as varchar(19)) LAB_RESULT_CM_ID
   when length(lab.RESULT_UNIT) > 11 then substr(lab.RESULT_UNIT, 1, 11)
   else trim(replace(upper(lab.RESULT_UNIT), '(CALC)', ''))
   end RESULT_UNIT
-, lab.NORM_RANGE_LOW
+, cast(lab.NORM_RANGE_LOW as varchar(10)) NORM_RANGE_LOW
 , case
   when lab.NORM_RANGE_LOW is not null and lab.NORM_RANGE_HIGH is not null then 'EQ'
   when lab.NORM_RANGE_LOW is not null and lab.NORM_RANGE_HIGH is null then 'GE'
   when lab.NORM_RANGE_LOW is null and lab.NORM_RANGE_HIGH is not null then 'NO'
   else 'NI'
   end NORM_MODIFIER_LOW
-, lab.NORM_RANGE_HIGH
+, cast(lab.NORM_RANGE_HIGH as varchar(10)) NORM_RANGE_HIGH
```

Comment:
I unintentionally deleted this field when refactoring for performance; not cool.